src/main/java/medicaldoctor/controllers/navbar/NavigationBarController.java

```diff
@@ -72,103 +81,123 @@
     private Button buttonLiverFunction;
     
     @FXML
-    void patientRegistration(ActionEvent event){
-        
+    void patientRegistration(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.PATIENT_REGISTRATION);
+        AppSession.cm.showScreen(LookUp.PATIENT_REGISTRATION);
     }
     
     @FXML
-    void patientLookup(ActionEvent event){
-        
+    void patientLookup(ActionEvent event) throws Exception{
+        AppSession.cm.load(LookUp.PATIENT_LOOKUP);
```

Comment:
rename `cm` to `CONTROLLER_MANAGER` or something more descriptive. don't use abbreviations.